vimeo-networking/src/main/java/com/vimeo/networking/model/notifications/NotificationType.java

```diff
@@ -35,4 +36,30 @@ public static NotificationType notificationTypeFromString(@NotNull String type)
         }
     }
 
+    @Override
+    public String toString() {
+        return stringFromNotificationType(this);
```

Comment:
I don't think this is a good idea. The default `toString` of an enum will return the name of the enum value. If the purpose of changing this is to return a serialized value of the enum, then we should create a different method than overriding toString.